config-stub.yml

```diff
@@ -0,0 +1,11 @@
+github-user: XXX
+github-password: XXX
+jira-server: https://tangle.atlassian.net
+jira-user: XXX
+jira-password: XXX
+jira-project: FRO
```

Comment:
Let's add two stub files, one for frontend and seconds for the backend, e.g. `config-stub.backend.yaml`:

```yml
github-user: XXX
github-password: XXX
jira-server: https://tangle.atlassian.net
jira-user: XXX
jira-password: XXX
origin-branch: master
jira-project: BAC
jira-release-project: RELEASE
jira-task-extra:
  component: Backend
release-set: echo
```

Reply:
Fixed.